docs/js/modal.md

```diff
@@ -49,11 +46,12 @@ Based on this markup, the component will get all content inside of `[data-modal]
 
 ## Options
 
-Modal provides three customizable options: `container`, `triggerClose` and `triggerOpen`. By default `.modal` is appended in the `body`, `triggerClose` and  `triggerOpen` are `null`.
+Modal provides some customizable options as: `container`, `size`, `triggerClose` and `triggerOpen`. The default of `.modal` is to be appended in the `body`, the `size` is 'medium', `triggerClose` and  `triggerOpen` are `null`.
 
 | Option            | Default | Description |
 |-------------------|-------------|
 | container  | `"body"` | A new string selector to append `.modal` |
+| size | `"medium"` | Modal sizes that can be "small", "large" or "medium" |
```

Comment:
Ajustar aqui também. Vê se assim ficaria melhor: `The size may vary between small, large, or medium`
Adicionando a vírgula antes do `or`